mantidproject/mantid
# Remove the `ISISIndirectEnergyTransferWrapper` algorithm

### Description of work
This PR removes the `ISISIndirectEnergyTransferWrapper` algorithm. This algorithm was introducted back in the days of MantidPlot in an attempt to fix an issue with the Energy Transfer tab of the Data Reduction interface, in which when reducing a batch of run numbers, the reduced file created for each subsequent run number would be slightly larger than the previous (#24582 ). At the time, this was assumed to have been caused by duplication of the algorithm history. I have checked on Mantid Workbench if each subsequent reduced file has a different size and this no longer appears to be the case when using the `ISISIndirectEnergyTransfer` algorithm. Therefore I would like to remove the `ISISIndirectEnergyTransferWrapper` algorithm.

*There is no associated issue.*

### To test:
1. `Interfaces`->`Indirect`->`Data Reduction`->`Energy Transfer` interface
2. Select IRIS instrument
3. Type in run numbers 83467-83472 and click Run once they are loaded
4. Check the size of each output file ending in `_red` (by using the arrow next to the workspace in the ADS). The sizes should be roughly the same as each other.


### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
